src/hooks/core/push-notification/usePushNotificationHook.ts

```diff
@@ -16,6 +16,7 @@ import useUserAuthHook from '../auth/useUserAuthHook';
 import TokenStorage, {ITokenEnum} from '../../../utils/storage/custom/tokenStorage';
 import {getAnonymousUserId, getUserId} from '../../../utils/users';
 import {getMessageDetail} from '../../../service/repo/messageRepo';
+import useChatScreenHook from '../../screen/useChatScreenHook';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useChatScreenHook` is added, but it's not clear if this hook is being used in the correct context. Hooks are meant to be used within React components or other hooks. If `usePushNotificationHook` is not a React component or another hook, this could lead to unexpected behavior.

```diff
+ import useChatScreenHook from '../../screen/useChatScreenHook';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->